src/InputWithLabel.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import {useRef, useEffect} from 'react'
+import PropTypes from 'prop-types'
+
+const InputWithLabel = ({id, value, onChange, children}) => {
+    const inputRef = useRef();
+
+    useEffect(()=>{
+        inputRef.current.focus();
+    },[value])

```

Comment:
the instructions are asking to not have a dependency array on this hook. In this case, the input field is getting re-focused with each keystroke, which may not matter to the common user, but is not ADA compliant (if using a screen reader, the reader will yell the input's label and value with every key stroke, not ideal)